src/main/java/net/masterthought/cucumber/Trends.java

```diff
@@ -30,10 +31,69 @@
 
     private long[] durations = new long[0];
 
+    private  FeatureScenario[][] featuresDetail = new FeatureScenario[0][0];
+
     public String[] getBuildNumbers() {
         return buildNumbers;
     }
 
+    public FeatureScenario[][] getFeaturesDetail() {
+    return featuresDetail;
+    }
+
+    public ArrayList<TrendTableRow> collectTrendFeatureScenario() {
+
+        ArrayList<TrendTableRow> trendTableRows = new ArrayList<>();
+
+        for (int i = 0 ; i < buildNumbers.length;i++){
+
+            for (int j=0;j<featuresDetail[i].length;j++){
+                FeatureScenario featureScenario = featuresDetail[i][j];
+                TrendTableRow trendTableRow = new TrendTableRow(featureScenario.getDeviceName(), featureScenario.getFeatureName(), featureScenario.getScenarioName(), featureScenario.getId());
+                if(!isAlreadyInTrenTableRows(trendTableRow,trendTableRows)){
+                    trendTableRows.add(trendTableRow);
+                }
+            }
+        }
+
+        for (int i = 0 ; i < buildNumbers.length;i++){
+            FeatureScenario[] featureScenarioPerBuild = featuresDetail[i];
+            for (TrendTableRow trendTableRow: trendTableRows) {
+                if (getStatus(trendTableRow, featureScenarioPerBuild)!=null){
+                    trendTableRow.setStatus(getStatus(trendTableRow, featureScenarioPerBuild));
+                }else {
+                    trendTableRow.setStatus("-");
```

Comment:
status has predefined values and this one does not look like one of them

Reply:
the idea from this part is to collect status of feature & scenario per build from trend json. I never changed the status here. I just collect from cucumber status. But in some case, we cannot find feature and scenario in trend json, then we mark is as '-'

Reply:
@suciista09 May be better to simply mark it as "UNKNOWN" ?